docs/source/developers/docker.rst

```diff
@@ -0,0 +1,224 @@
+.. raw:: html
+
+   <!--
```

Comment:
Can't you copy/paste the copyright block from another .rst file instead? Using raw HTML for this is weird.

Reply:
Updating.